syntax = "proto3";

package Phone;

message Phone
{
    float timestamp = 1;
    bytes color_img = 2;
    repeated int32 depth_img = 3;
    int32 depth_width = 4;
    int32 depth_height = 5;
    repeated float pose = 6;
}